docs/reference/logging/awslogs.md

```diff
@@ -34,9 +34,10 @@ You can use the `--log-opt NAME=VALUE` flag to specify Amazon CloudWatch Logs lo
 
 ### awslogs-region
 
-You must specify a region for the `awslogs` logging driver. You can specify the
-region with either the `awslogs-region` log option or `AWS_REGION` environment
-variable:
+The `awslogs` logging driver must send your Docker logs to a specified region;
```

Comment:
The phrasing is a bit odd.  Maybe;

The `awslogs` logging driver sends your Docker logs to a specific region. Use the `awslogs-region` log option or the `AWS_REGION` environment variable to
set the region value. By default, if your Docker daemon is running on an EC2 instance and no region is set, the driver uses the instance's region.